SingularityService/src/test/java/com/hubspot/singularity/scheduler/SingularitySchedulerTest.java

```diff
@@ -3841,6 +3842,12 @@ public void testCronScheduleChanges() throws Exception {
     );
   }
 
+  @Test
+  public void testBrokenQuartzCron() throws Exception {
+    String quartz = "0 0/60 * * * ?";
+    new CronExpression(quartz);
+  }
+
```

Comment:
Not sure how useful this test is in the sense that this is technically not a valid expression, but library maintainers have made it work in the past and since decided to change that behavior. Maybe add a note about the quartz upgrade for background here in case this starts failing on a future upgrade?